Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -114,21 +114,39 @@ struct HitData {
                  x, y, z, t);
 };
 
+struct MeasurementData {
+  /// Event-unique measurement identifier. Each value can appear at most once.
+  uint64_t hit_id;
```

Comment:
This should probably be `measurement_id`.

Reply:
Done.